ert_gui/simulation/models/base_run_model.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, queue_config, phase_count=1):
         self._run_context = None
         self._last_run_iteration = -1
         self.reset( )
+        self._ensemble_size = 0
```

Comment:
Please rename to `_initial_active_size` to avoid confusion with the ensemble size. Also, this seems to be duplicated data. Could we implement `_initial_active_size` as a `property` returning `self.initial_realziations_mask.count(True)` instead?